tpcc/payment.go

```diff
@@ -175,29 +175,28 @@ func (p payment) run(db *sql.DB, wID int) (interface{}, error) {
 			}
 
 			// Update customer with payment.
+			// If the customer has bad credit, update the customer's C_DATA and return
+			// the first 200 characters of it, which is supposed to get displayed by
+			// the terminal. See 2.5.3.3.
```

Comment:
> See 2.5.3.3

2.5.2.2?

Reply:
2.5.3.3 has the part that says "just return 200 characters", which is what i was getting at.